app/assets/stylesheets/components/static_pages.scss

```diff
@@ -0,0 +1,77 @@
+.static_page {
+  margin-top: -48px;
+  .sub_nav {
+    background-color: transparent;
+    position: static;
+    background-color: $static-page-heading-background;
+  }
+  h1.static_page_heading {
+    @include title36();
+    color: $static-page-heading-text;
+    text-align: center;
+    padding: 50px $baseline 100px $baseline;
+    margin: 0;
+    background-color: $static-page-heading-background;
+  }
+  .static_page_content {
+    max-width: $site_max_width;
+    box-sizing: border-box;
+    padding: 0 $baseline * 1.5;
+    margin-left: auto;
+    margin-right: auto;
+    @media #{$mobile} {
+      padding-left: 0;
+      padding-right: 0;
+    }
+    .static_page_content_inner {
+      background-color: $static-page-content-background;
+      position: relative;
+      top: -$baseline * 1.5;
+      @include shadow2();
+      border-radius: 6px;
+      padding-bottom: $baseline;
+      h2 {
+        &::before {
+          position: absolute;
+          left: -38px;
+          content: image-url("svg/icon_static_page_anchor_link.svg");
+          width: 24px;
+          height: 24px;
+
+        }
+        @include title24();
+        color: $static-page-h2-text;
+        box-shadow: none;
+        border-radius: 0;
+        line-height: 1.3;
+        padding: $baseline * (7/3) $baseline * 2 0 $baseline * 2;
+        @media #{$mobile} {
+          padding-left: $baseline;
+          padding-right: $baseline;
+        }
+        margin-top: $baseline * 2;
+        border-top: 2px solid $static-page-content-seperator;
+        &:first-child {
+          border-top: 0;
+          margin-top: 0;
+        }
+      }
+      p {
+        @include paragraph18();
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
```

Comment:
@ofsound Could you experiment a bit with these? I think this rendering setting actually does make sense on retina osx — however it will be blurrier on non-retina screens. Perhaps there's a way to restrict? 

@smoofles recommended this:

``` 
* {
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
``` 

That also fixes headers to be a bit more sane and figma-compatible. However, there could be some styles that appear too lightly like "back to artist" — maybe you could take the temperature..